onnxruntime/core/providers/cuda/generator/constant_of_shape.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/data_types.h"
+#include "core/framework/op_kernel.h"
+#include "core/providers/cuda/cuda_common.h"
+#include "core/providers/cpu/generator/constant_of_shape.h"
+#include "core/providers/cuda/shared_inc/cuda_utils.h"
+
+namespace onnxruntime {
+namespace cuda {
+
+class ConstantOfShape final : public ConstantOfShapeBase, public OpKernel {
```

Comment:
>ConstantOfShap [](start = 6, length = 14)

Do we want to explicitly prohibit copy/assignment?

Reply:
will do

---
In reply to: [290988684](https://github.com/microsoft/onnxruntime/pull/1168#discussion_r290988684) [](ancestors = 290988684)